test/spec/services/maprules.js

```diff
@@ -163,7 +163,17 @@ describe('maprules', function() {
     });
     describe('#clearRules', function() {
         it('clears _validationRules array', function() {
+            iD.serviceMapRules.clearRules();
+            expect(iD.serviceMapRules.validationRules()).to.be.empty;
+
+            iD.serviceMapRules.addRule({
+                geometry:'node',
+                equals: {amenity:'marketplace'},
+                absence:'name',
+                warning:'\'Marketplace\' preset must be coupled with name'
+            });
```

Comment:
previously, this test case only passed because state from the previous test was leaking into this one. Now it explicitly sets up the environment.